content/en/docs/installation/api_compatibility.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "API compatibility"
+linkTitle: "API compatibility"
+weight: 70
+type: "docs"
+---
+
+cert-manager aims to abide by the same API compatibility policy as upstream Kubernetes APIs, see [Kubernetes Deprecation Policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-parts-of-the-api).
+The main purpose of this is to ensure a smooth upgrade and downgrade experience for users, i.e to make sure that users' cert-manager custom resources keep functioning as before after an upgrade or downgrade of cert-manager.
```

Comment:
```suggestion
The main purpose of this is to ensure a smooth upgrade and downgrade experiences for users, i.e to make sure that users' cert-manager custom resources keep functioning as before after an upgrade or downgrade of cert-manager.
```

Reply:
Is this suggestion not grammatically incorrect? I think this section is fine as-is.

Reply:
Hmm, indeed, it's a half-step.
```suggestion
The main purpose of this is to ensure smooth upgrade and downgrade experiences for users, i.e to make sure that users' cert-manager custom resources keep functioning as before after an upgrade or downgrade of cert-manager.
```

The problem I have with the as-is content is that one can't both upgrade and downgrade concurrently, so it isn't one single experience. One can have other experiences.